app/client/keybase/keystore.go

```diff
@@ -47,6 +49,85 @@ func NewKeybaseInMemory() (Keybase, error) {
 	return &badgerKeybase{db: db}, nil
 }
 
+type yamlConfig struct {
+	ApiVersion string            `yaml:"apiVersion"`
+	Kind       string            `yaml:"kind"`
+	MetaData   map[string]string `yaml:"metadata"`
+	Type       string            `yaml:"type"`
+	StringData map[string]string `yaml:"stringData"`
+}
+
+// Creates/Opens the DB and initialises the keys from the YAML file
+// FOR DEV/LOCANET PURPOSES ONLY
+func InitialiseKeybase(path string) (Keybase, error) {
```

Comment:
I'd prefer to refactor this a bit. We are doing two things here:

1. Initialising a debug keybase for DEV purposes
2. Retrieving a keybase for PROD purposes

My request would be to move `InitialiseDebugKeybase` as a helper inside one of the `init` functions so it's a one-time check. Simply refer to that keybase in the normal code flow.

Reply:
@Olshansk I have added a `debug` folder inside the keybase and am initialising the DB at `$HOME/.pocket/keys` for the debug DB so this can be done in the `init()` function otherwise there would be no way to use `init()` with a path that could change.

Reply:
It's a step in the right direction, but was hoping to take it further.

At the moment, every function in our PROD CLI starts with a `NewDebugKeybase()` and I think we can make it really prod-ready in this PR.

```
		RunE: func(cmd *cobra.Command, args []string) error {
			// Open the debug keybase at $HOME/.pocket/keys
			kb, err := debug.NewDebugKeybase()
```

The `init` function gets called when we import something. What I'm thinking is:

1. Create a file that contains an anonymous import of the debug package
2. By importing it, we immediately load the debug DB if its empty (we can exclude this from production binaries)
3. Replace the `kb` assignment with a prod ready assignment